libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1322,7 +1323,11 @@ void ExpressionCompiler::appendArithmeticOperatorCode(Token::Value _operator, Ty
 {
 	bool c_isSigned;
 	bool c_isFractional;
-	u256 c_shift = u256(1);
+	u256 c_fractionShift = u256(1);
```

Comment:
all the others had `c_` attached...I figured it was a naming convention for this function.